app/models/zone.rb

```diff
@@ -1,2 +1,7 @@
 class Zone < ApplicationRecord
+  DEFAULT_ZONE_CERC_ID = 29
```

Comment:
Are we happy for this to just be documented in the test file, and the commit history? Maybe it would grease the wheels a little to call it `DEFAULT_ZONE_SOUTHWARK_CERC_ID`?

Reply:
good idea, thks